### DISCUSSION TITLE
How to create a subset according to the texture component in pole figure?

### DISCUSSION THREAD

**Discussion by caoyucheng-1** [mtex-toolbox/mtex#1425] [Ask Anything] [answered]
Hi everyone, 

I' m interested in how to create a subet according to the texture component in PF.  I achieved this way in AztecCrystal but I want to know how to achieve it in MTEX.

![image](https://user-images.githubusercontent.com/75624147/167296751-3b129f57-80ed-4aae-a3ca-742bd9cf5849.png)
![image](https://user-images.githubusercontent.com/75624147/167296758-dd7f71ad-ce37-4fd9-8760-f84604b9181a.png)

**Comment by kilir** [comment, marked as answer]
Hi,
maybe you are looking for something like this?
```
mtexdata titanium
ebsd=ebsd('t')

% crystal directiosn in specimen coords
c_axis_direction = ebsd('t').orientations * Miller(0,0,0,1,ebsd('t').CS)
c_axis_direction.antipodal = 1
% some reference direction (could be anything)
ref_dir = zvector;

% a threshold
th = 25 * degree
a_d = angle(ref_dir,c_axis_direction);
cond = a_d < th;

plot(ebsd(~cond))
hold on
plot(ebsd(cond),a_d(cond))
hold off

nextAxis
plot(c_axis_direction(~cond),'MarkerSize',2)
hold on
plot(c_axis_direction(cond),a_d(cond),'MarkerSize',2)
circle(ref_dir,th)
hold off
```
Cheers,
Rüdiger

**Reply by caoyucheng-1** [reply]
Thank you very much! It helps me a lot.
